images/pack-builder/stack-base.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM oryxdevms/build:latest
+
+LABEL io.buildpacks.stack.id=com.microsoft.oryx.stack
+
+# Configure non-root user
+RUN groupadd -g 1002 oryx_group && \
+	useradd -u 1001 -g oryx_group oryx_user && \
```

Comment:
It is easier to read when the commands are all indented, so it looks like a script